src/javascripts/ng-admin/Crud/field/maEmbeddedListField.js

```diff
@@ -59,7 +59,7 @@ export default function maEmbeddedListField() {
         <div class="remove_button_container">
             <a class="btn btn-default btn-sm" ng-click="remove(entry)"><span class="glyphicon glyphicon-minus-sign" aria-hidden="true"></span>&nbsp;<span translate="REMOVE"></span></a>
         </div>
-        <div class="form-field form-group" ng-repeat="field in ::fields track by $index">
+        <div ng-repeat="field in ::fields track by $index" compile="::field.getTemplateValueWithLabel(entry)">
```

Comment:
The "form-field" and "form-group" classes here are redundant, since "ma-field" inside this div also creates a div with those classes. Having them repeated here interferes with hiding rows based on entity values.